README.md

```diff
@@ -94,6 +94,46 @@ Options:
 
 Like everything in speccy, this only works for OpenAPI v3.
 
+### Calling Speccy from Code
+
+Not just a command line tool, speccy can be used to normalize machine-readable specifications.
+
+
+The loader object will return a promise that resolves to an object containing
+the specification.  For example:
+
+```javascript
+const loader = require('speccy/lib/loader');
+
+const options = {
+  resolve: true,   // Resolve external references 
+  jsonSchema: true // Treat $ref like JSON Schema and convert to OpenAPI Schema Objects
+};
+
+loader
+  .loadSpec('path/to/my/spec', options)            // Load the spec...
+  .then(spec => console.log(JSON.stringify(spec)); // ...and print it out.
+```
+
+If `options.resolve` is truthy, speccy will resolve _external_ references.
+However, the loaded specification it may contain JSON pointers in lieu of
+_internal_ javascript references.  A library like reftools can convert those
```

Comment:
What are javascript references? Are we talking about JSON Pointers? If so we should link to them somehow, or https://swagger.io/docs/specification/using-ref/#escape-characters or the appropriate RFC wherever that is. 

What would we need reftools for? Could you show examples of the difference to explain why and when the two approaches make sense?

Reply:
@tonious bump!

Reply:
I think a Javascript reference here refers to an in-memory object which refers to itself (or part of itself) directly, i.e. the object-model of an OAS document that has already been de-referenced.